### DISCUSSION TITLE
GDL icon

### DISCUSSION THREAD

**Discussion by PattiMichelle** [gnudatalanguage/gdl#885] [Ideas]
Not a very important idea - but should GDL have an icon?  I'm thinking for those of us still using GUI Linux interfaces (gnome, kde, etc.) click a distinctive icon to invoke the GDL IDE.

**Comment by pjb7687** [comment]
I think that would be awesome.

**Reply by PattiMichelle** [reply]
I'm not a graphic-arts type, or I would propose one.  Maybe a picture of the globe?  Here's a particular map projection that I _really_ like (although I don't know the name of the projection - oops, yes I do - the Atlantis projection).  I did make the background transparent, 'cause I know that looks pretty good pinned to a taskbar...
![GDL_Pre-icon_1](https://user-images.githubusercontent.com/16847444/107163520-2a783100-6967-11eb-9326-9f8d5d55cded.png)
![GDL_Pre-icon_2](https://user-images.githubusercontent.com/16847444/107163524-2fd57b80-6967-11eb-90ab-f8b196b30c8b.png)
(Also, for some reason uploading the .png files to github removes the alpha channel...)

These might be a poor aspect ratio (256x128) - maybe a different map projection that will yield a more square aspect ratio?

![GDL_Pre-icon_3](https://user-images.githubusercontent.com/16847444/107163675-1aad1c80-6968-11eb-952f-0f863f1abfde.png)

![GDL_Pre-icon_4](https://user-images.githubusercontent.com/16847444/107163804-c5253f80-6968-11eb-95fc-1631ee1d7514.png)

**Comment by slayoo** [comment]
What about reusing the icon we have for github repo:

![image](https://user-images.githubusercontent.com/89685/107163618-ddb64980-69aa-11eb-8b1c-f3ee36038993.png)

**Reply by slayoo** [reply]
here's the url: https://avatars.githubusercontent.com/u/8539759 (or with scaling: https://avatars.githubusercontent.com/u/8539759?s=100)

**Comment by pjb7687** [comment]
Thanks for the suggestions, but what I expected was something more like a pictogram (+1 if vector graphics) rather than a shrunk image. As @slayoo mentioned, we already have an image-based 'icon'.

**Reply by alaingdl** [reply]
recycling the Debian Astro Logo just by changing the line below ?
you can see it by doing
```
wget http://ftp.fr.debian.org/debian/pool/main/d/debian-astro/debian-astro-logo_2.2_all.deb
dpkg -x debian-astro-logo_2.2_all.deb .
```
![GDL-logo-100x155](https://user-images.githubusercontent.com/38264661/107287892-e702ed00-6a62-11eb-800f-6b897f16fe98.png)

Yes this PNG comes from a SVG ... (to be done ...)

**Reply by alaingdl** [reply]
(abviously we will have to check we can do that in respect with the copyright)
